src/main/java/teammates/ui/template/FeedbackResponseRow.java

```diff
@@ -73,6 +78,15 @@ public FeedbackResponseRow(int fbIndex, int personIndex, String personType,
         }
     }
 
+    private String getNewResponseText(FeedbackResponseAttributes response, FeedbackQuestionAttributes question,
+                                      FeedbackSessionResultsBundle results) {
+        response.giver = response.recipient;
+        response.giverSection = response.recipientSection;
+
+        String responseText = results.getResponseAnswerHtml(response, question);
+        return "No Response" + responseText.substring(responseText.indexOf("</span>") + "</span>".length());
```

Comment:
```java
responseText.substring(responseText.indexOf("</span>") + "</span>".length())
```
What are you trying to achieve here?

Reply:
Basically responseText returns `<span class="color-neutral"></span><span>&nbsp;&nbsp;[Perceived Contribution: <span class="color-positive">Equal Share +20%</span>]</span>` for a given `Perceived Contribution`. The span with color-neutral class is the default response returned when there is no response from the given user and has no text . 

So I took the back part which is `<span>&nbsp;&nbsp;[Perceived Contribution: <span class="color-positive">Equal Share +20%</span>]</span>`

Ultimately, it would not make a difference to the UI shown even without the substring part.